packages/turbo-benchmark/src/ttft/build-vega.ts

```diff
@@ -0,0 +1,234 @@
+import fetch from "node-fetch";
+import vega from "vega";
+import { put } from "@vercel/blob";
+import sharp from "sharp";
+import type { TTFTData } from "../helpers";
+
+const TINYBIRD_AGGREGATE =
+  "https://api.us-east.tinybird.co/v0/pipes/turborepo_perf_ttft_all.json";
+
+const PNG_FILENAME = "aggregate.png";
+
+async function fetchAggregateFromTinybird(): Promise<{
+  data: Array<TTFTData>;
+}> {
+  const token = process.env.TINYBIRD_TOKEN;
+  const resp = await fetch(`${TINYBIRD_AGGREGATE}?token=${token}`);
+  return resp.json() as Promise<{ data: Array<TTFTData> }>;
+}
+
+function generateVegaSpec(aggregateData: { data: Array<TTFTData> }): vega.Spec {
+  const source0: vega.ValuesData = {
+    name: "source_0",
+    values: aggregateData.data,
+    transform: [
+      {
+        type: "formula",
+        expr: "datum.durationMicroseconds/1000",
+        as: "durationMs",
+      },
+    ],
+  };
+  const vegaConfig: vega.Spec = {
```

Comment:
Would love a comment here about where you got this / how to generate it again if needed.